documentation/documentation/docs/projectform/project_upload_form.md

```diff
@@ -0,0 +1,34 @@
+# Project upload guide
+A teacher can upload a new project of their course by going to the url ```projects/create``` where they will need to fill out data corresponding to the project they wants to upload.
```

Comment:
No need for triple backticks

`/projects/create` *

Reply:
https://github.com/SELab-2/UGent-3/pull/250/commits/a175039634b28c72360c862b6f82fb2f88d1afe0